Polyfill.md

```diff
@@ -70,6 +70,10 @@ Some divergences that we've identified as potentially desirable:
   standard behavior:
   - Return zero when conversion from floating point to integer fails;
   - Optionally canonicalize NaN values.
+* **[NaN bit-pattern propagation](AstSemantics.md#floating-point-operators)**:
+  Regardless of WebAssembly behavior, an asm.js polyfill will follow its
```

Comment:
Hmm, that's unfortunate: I think the polyfill should detail what's different because JS semantics are that way, versus what's different because asm.js' subset of JS is that way. e.g. the alignment stuff could "just work" with JS but perf would suffer because asm.js.